05.py

```diff
@@ -31,7 +31,7 @@
 df.createOrReplaceTempView('GROUPED')
 sqlContext.registerJavaFunction("ws", "io.warp10.spark.WarpScriptUDF2", "BINARY")
 
-df = sqlContext.sql("SELECT ws('@04.mc2', _2) AS wrapper FROM GROUPED")
+df = sqlContext.sql("SELECT ws('@05.mc2', _2) AS wrapper FROM GROUPED")
```

Comment:
The loading should have been adapted from 04.py so the types are identical and therefore no need for a duplicate 05.mc2